torch_xla/csrc/tensor_methods.cpp

```diff
@@ -692,6 +692,15 @@ XLATensor XLATensor::elu(const XLATensor& input, at::Scalar alpha,
       ir::ops::Elu(input.GetIrValue(), alpha, scale, input_scale));
 }
 
+XLATensor XLATensor::elu_backward(const XLATensor& grad_output,
```

Comment:
I'm thinking ... this should be after elu_?

Reply:
I've followed the order in `AtenXlaTypeBase`. Why should it be after `elu_`?

Reply:
For the same reason 'abc' > 'ab'?

Reply:
Done, alphabetical order sounds good to me. If we stick to it though, we will have unrelated methods brought together and related methods scattered around.

Reply:
I prefer that, vs. trying to figure out the groups ;)